creator/studies/dataservice/utils.py

```diff
@@ -0,0 +1,104 @@
+import logging
+from pprint import pformat
+
+import requests
+from d3b_utils.requests_retry import Session
+from kf_utils.dataservice.scrape import yield_kfids
+
+# DO NOT RE-ORDER - needed for deletion in StudyGenerator.clean
+ENTITY_ENDPOINTS = {
+    "read_group": "read-groups",
+    "read_group_genomic_file": "read-group-genomic-files",
+    "sequencing_experiment": "sequencing-experiments",
+    "sequencing_experiment_genomic_file": "sequencing-experiment-genomic-files",  # noqa
+    "genomic_file": "genomic-files",
+    "biospecimen_genomic_file": "biospecimen-genomic-files",
+    "biospecimen": "biospecimens",
+    "outcome": "outcomes",
+    "phenotype": "phenotypes",
+    "diagnosis": "diagnoses",
+    "participant": "participants",
+    "family_relationship": "family-relationships",
+    "family": "families",
+    "study": "studies",
+}
+ENDPOINTS = [v for k, v in ENTITY_ENDPOINTS.items()]
+
+logger = logging.getLogger(__name__)
+
+
+def delete_kfids(host, endpoint, kfids):
```

Comment:
Both of these delete methods will be removed once https://github.com/kids-first/kf-utils-python/pull/24 is released and imported from `kf-utils-python`